devito/functions/basic.py

```diff
@@ -27,24 +27,20 @@
 class Basic(object):
     """
     Three relevant types inherit from this class: ::
-
```

Comment:
watch out: you're destroying the docstrings by removing these blank lines. Double check this everywhere.